tests/external_config/test_external_command.py

```diff
@@ -78,33 +78,34 @@ def test_properties(self):
         """
         Make sure public properties can be accessed without exception
         """
-        self.assertEquals(
+        self.assertEqual(
             self.ec.pipeline_configuration_name, self._pipeline_configuration_id
         )
         # Yes, the prop. name and dict key are different
-        self.assertEquals(self.ec.system_name, self.ec_data["callback_name"])
-        self.assertEquals(self.ec.engine_name, self.ec_data["engine_name"])
-        self.assertEquals(self.ec.display_name, self.ec_data["display_name"])
-        self.assertEquals(self.ec.group, self.ec_data["group"])
-        self.assertEquals(self.ec.icon, self.ec_data["icon"])
-        self.assertEquals(self.ec.is_group_default, self.ec_data["group_default"])
+        self.assertEqual(self.ec.system_name, self.ec_data["callback_name"])
```

Comment:
These generated warnings in pytest because they are deprecated, so I fixed the calls.

Reply:
You can skip this file, it's all about fixing that warning.